components/StudentResource.jsx

```diff
@@ -0,0 +1,124 @@
+import Image from "next/image";
+import Link from "next/link";
+import styled from "styled-components";
+import PropTypes from "prop-types";
+
+import { Text } from "./Text";
+import { Button } from "./Button";
+import { media } from "../utils";
+
+export const StudentResource = ({
+  titleText,
+  descriptionText,
+  buttons,
+  graphic,
+  graphicWidth,
+  graphicHeight,
+  isGraphicLeft = true,
+  ...props
+}) => (
+  <Wrapper isGraphicLeft={isGraphicLeft} {...props}>
+    <Graphic
+      src={graphic}
+      width={graphicWidth}
+      height={graphicHeight}
+      isGraphicLeft={isGraphicLeft}
+    />
+    <InfoSection isGraphicLeft={isGraphicLeft}>
+      <Title>{titleText}</Title>
+      <Text>{descriptionText}</Text>
+      <ButtonSection isGraphicLeft={isGraphicLeft}>
+        {buttons.map((button) => (
+          <Link key={button.text} href={button.link}>
+            <a>
+              <SButton backgroundColor={button.color}>{button.text}</SButton>
+            </a>
+          </Link>
+        ))}
+      </ButtonSection>
+    </InfoSection>
+  </Wrapper>
+);
+
+const Title = styled.h2`
+  ${({ theme }) => `
+      font-family: ${theme.font.josefin};
+      color: ${theme.colors.black};
+  `};
+  margin: 0;
+`;
+
+const Graphic = styled(Image)`
+  /* TODO: curved border ? */
```

Comment:
we typically use ```44px``` so feel free to add that now if u want (or later is fine too haha)

Reply:
Will add later